src/components/widgets/accounts-invoices-list/accounts-invoices-list.directive.coffee

```diff
@@ -52,6 +51,7 @@ module.controller('WidgetAccountsInvoicesListCtrl', ($scope, $q, ImpacRoutes, Bo
   # --------------------------------------
   w.initContext = ->
     $scope.isDataFound = w.content?
+    $scope.trxList.resources = w.content.invoice_endpoint
```

Comment:
You should not rely on what the api will return to tell your frontend component that it will receive invoices / the previous way seemed fine to me